commands/src/main/java/org/wildfly/extras/creaper/commands/infinispan/cache/RemoveCache.java

```diff
@@ -4,11 +4,17 @@
 import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
 import org.wildfly.extras.creaper.core.online.operations.Address;
 import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
 
 public final class RemoveCache implements OnlineCommand {
     private final Address address;
+    private final boolean allowResourceServiceRestart;
 
     public RemoveCache(String cacheContainer, CacheType cacheType, String cacheName) {
+        this(cacheContainer, cacheType, cacheName, false);
+    }
+
+    public RemoveCache(String cacheContainer, CacheType cacheType, String cacheName, boolean allowResourceServiceRestart) {
```

Comment:
Or possibly this could be automatic, if we detect the server isn't able to remove a cache without service restart (version check).

I know this goes against "restarts should be explicit", but it's an option worth considering. WDYT?